CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # Changelog
 
+## v0.20.0 (2018-5-11)
+
+### Request Hooks: Intercepting HTTP requests ([#1341](https://github.com/DevExpress/testcafe/issues/1341))
+
+TestCafe now allows you to record HTTP request data or mock responses. You can also create a custom HTTP request hook to emulate authentications like  **Kerberos** or **Client Sertificate Authentication**.
+
+See [Intercepting HTTP Requests](docs/articles/documentation/test-api/intercepting-http-requests/README.md) for more information.
+
+### Enhancements
+
+#### :gear: Specifying resources accessed bypassing a proxy server ([#1791](https://github.com/DevExpress/testcafe/issues/1791))
+
+TestCafe now allows you to bypass the proxy server when accessing specific resources.
+
+To specify resources that require direct access, use the [--proxy-bypass](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--proxy-bypass-rules) flag in the command line or the [useProxy](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.md) API method's parameters.
+
+```sh
+testcafe chrome my-tests/**/*.js --proxy proxy.corp.mycompany.com --proxy-bypass localhost:8080,internal-resource.corp.mycompany.com
+```
+
+```js
+runner.useProxy('172.0.10.10:8080', ['localhost:8080', 'internal-resource.corp.mycompany.com']);
+```
+
+#### :gear: Specifying testing metadata ([#2242](https://github.com/DevExpress/testcafe/issues/2242))
+
+TestCafe allows you to specify additional information for tests in the form of key-value metadata and use it in reports.
+
+You can define metadata for a fixture or a test using the [meta](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.md#specifying-testing-metadat) method:
+
+```js
+fixture 'MyFixture'
+    .meta('fixtureID', 'f-0001')
+    .meta({ author: 'John', creationDate: '05/03/2018' });
+```
+
+```js
+test
+    .meta('testID', 't-0005')
+    .meta({ severity: 'critical', testedAPIVersion: '1.0' });
+    ('MyTest', async t => { /* ... */});
+```
+
+To include testing metadata to reports, use [custom reporter methods](https://devexpress.github.io/testcafe/documentation/extending-testcafe/reporter-plugin/reporter-methods.html).
+
+#### :gear: Passing a regular promise to `t.expect` ([#2207](https://github.com/DevExpress/testcafe/issues/2207))
```

Comment:
IMHO, header should say the opposite, like
```Passing a regular promise to t.expect is deprecated now```

Previously you could pass a regular promise without any problem. Now we throw an error in this case, and you should set a special flag to get previous behavior.